Bottom: d22154425d3f1cbf6ee9d33b9ac41249777bd0bc
Top:    20b8f8d07d76907c4aa9485d4e56f379ed161174
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-19 15:05:38 +0100

Refresh of hid-gtk-refactor-viewport-hand

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index e6c54d5..c3cd8da 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -62,6 +62,12 @@
 #define	FROM_PCB_UNITS(v)	coord_to_unit (Settings.grid_unit, v)
 #define	TO_PCB_UNITS(v)		unit_to_coord (Settings.grid_unit, v)
 
+//#define	DRAW_X(x)	(gint)(FLIP_X(x - gport->view_x0) / gport->zoom)
+//#define	DRAW_Y(y)	(gint)(FLIP_Y(y - gport->view_y0) / gport->zoom)
+
+//#define	VIEW_X(x)	(FLIP_X(x) * gport->zoom + gport->view_x0)
+//#define	VIEW_Y(y)	(FLIP_Y(y) * gport->zoom + gport->view_y0)
+
 /*
  * Used to intercept "special" hotkeys that gtk doesn't usually pass
  * on to the menu hotkeys.  We catch them and put them back where we
